### PR TITLE
Update build script

### PR DESCRIPTION
**Pull request by annakocheshkova**
Build script should download uwp package from public repo. 
The old logic is kept untouched. 
To download the packages from a private repo, use environment variable `NUGET_PRIVATE = "Yes"`.

### CODE REVIEW

File: build.cake [hunk 1]
```diff
@@ -223,59 +223,89 @@ Task("Externals-Uwp")
...
-            // them to AnyCPU automatically
-            contentPathSuffix = "runtimes/win10-" + module.NativeArchitectures[0] + "/" + contentPathSuffix;
-        }
-
-        // Move the files to the proper location
-        var files = GetFiles(tempContentPath + contentPathSuffix + "*");
-        MoveFiles(files, destination);
-    }
+    if (EnvironmentVariable("NUGET_PRIVATE") != "Yes") {
+		var packageSource = "https://www.nuget.org/api/v2/";
```
Comment (max-mironov) [inline build.cake]: Have you considered about moving this two parts to separate functions to make the overall method size a smaller + the name of the func will show the intent in a more clear and concise way?

File: build.cake [hunk 2]
```diff
@@ -223,59 +223,89 @@ Task("Externals-Uwp")
...
-            // Even though they are architecture dependent, Unity converts
-            // them to AnyCPU automatically
-            contentPathSuffix = "runtimes/win10-" + module.NativeArchitectures[0] + "/" + contentPathSuffix;
-        }
-
-        // Move the files to the proper location
-        var files = GetFiles(tempContentPath + contentPathSuffix + "*");
-        MoveFiles(files, destination);
-    }
+    if (EnvironmentVariable("NUGET_PRIVATE") != "Yes") {
```
Comment (max-mironov) [inline build.cake]: Does this env var documented somehow/somewhere? 

Also may be make sense to do smth like this: 
```
var usePrivateNuget = EnvironmentVariable("NUGET_PRIVATE")
if (string.IsNullOrEmpty(usePrivateNuget) || usePrivateNuget.toLowerCase() != "yes") 
```

File: build.cake [hunk 3]
```diff
@@ -223,59 +223,89 @@ Task("Externals-Uwp")
...
+			if (module.Moniker == "Crashes")
+			{
+				Warning("Skipping 'Crashes' for UWP.");
+				continue;
+			}
+			Information("Downloading " + module.DotNetModule + "...");
+			// Download nuget package
+			var package = repository.FindPackage(module.DotNetModule, SemanticVersion.Parse(UwpSdkVersion));
+			IEnumerable<IPackage> dependencies;
+			dependencies = new [] { package };
```
Comment (max-mironov) [inline build.cake]: May be join this and above line?

File: build.cake (hunk 1 above)
Comment (max-mironov) [reply build.cake]: @annakocheshkova after looking a little bit more it seems both code path (if .. else) contains a lot of similar logic, so please look at if we can use even ONE function for this with some switch inside it to decide if we dl from public or not e.g. smthg like this:
`downloadAndExtractNugets(usePublicFeed: bool)`
Not sure if naming for function is totally good :)

File: build.cake (hunk 2 above)
Comment (MatkovIvan) [reply build.cake]: You can use `EnvironmentVariable<bool>("NUGET_PRIVATE", false)` if the value is `true` instead of `Yes`

**Comment by max-mironov** [comment]
@annakocheshkova please fix formatting.
One item to discuss: we can omit 'NUGET_PRIVATE' env var at all and replace it with the following check:
`var usePublic = (!nugetUser || !nugetPassword || !nugetFeedId)`
@MatkovIvan what do you think about it?

Otherwise LGTM.

**Comment by MatkovIvan** [comment]
@max-mironov 👍 
@annakocheshkova It seems to differ should only downloading but not the logic of unpacking

**Review by max-mironov — DISMISSED** [review]
Formatting issues still exists (seems 4 spaces instead of 2), the rest LGTM

**Review by MatkovIvan — APPROVED** [review]

**Pull request merged**